castervoice/apps/vscode.py

```diff
@@ -300,7 +304,7 @@ class VSCodeNonCcrRule(MergeRule):
         "shift group left":
             R(Key("c-k, left"),
               rdescript="Shift Current Group of Tabs to the Left E.g. Swap with Pane to the Left"),
-        "shift group left":
+        "shift group right":
             R(Key("c-k, right"),
```

Comment:
Can you change the rdescript styles to `VS Code: My description`?

Reply:
With the recent changes rdescript is no longer necessary unless the command does not reflect the action it produces.